R/jointRPCAutils.R

```diff
@@ -0,0 +1,557 @@
+#OPTSPACE FUNCTION AND ITS AUXILIARY FUNCTIONS
+
+.optspace  <- function(x, ropt = 3, niter = 5, tol = 1e-5, verbose = FALSE){
```

Comment:
Just to check: do we need separate optspace function here, or could we use those in vegan? (including vegan internal functions).